src/tcgwars/logic/impl/gen7/CosmicEclipse.groovy

```diff
@@ -3707,8 +3707,18 @@ public enum CosmicEclipse implements LogicCardInfo {
             energyCost Y, C, C
             onAttack {
               damage 80
-              // TODO: edit it to include only tag team supporters
-              if(bg.em().retrieveObject("last_supporter_play_turn") == bg.turnCount) damage 80
+              if(bg.em().retrieveObject("last_supporter_play_turn") == bg.turnCount && bg.em().retrieveObject("last_tag_team_supporter_play_turn") == bg.turnCount) {
```

Comment:
Is the supporter turn check needed? If a tag team supporter was played, a supporter would've been played right?

Reply:
Yes. That was left over from when I was planning on just storing a true value in an effect and then setting it back to null when unregistering it.